README.md

```diff
@@ -1,53 +1,60 @@
 # Neutrinet VPN Certificates renewer
 
-This repository store 2 python scripts to renew your Neutrinet VPN certificate.
-One as to be runned on an Internet Cube that has vpnclient installed, the other
-one is to be runned in a standalone fashion.
+This repository stores a python3 script to renew your Neutrinet VPN certificate.
 
-**Remark**: this repository is used by the [neutrinet_ynh](https://github.com/Neutrinet/neutrinet_ynh) app, so don't rename/delete it unless you know what you are doing.
+You can run this script from your own computer.
 
-# Usage
+You can also run this script from your internet cube.
+Note that there is a [Yunohost app called Neutrinet](https://github.com/Neutrinet/neutrinet_ynh) just for that. 
+It will setup a daily cron task that will automatically renew your certificate when needed.
 
-## Standalone version
+**Warning**: As it is used by the Yunohost app, do NOT rename or delete the script unless you know what you are doing.
 
-Installation:
+## Installation
 
-    git clone https://github.com/neutrinet/renew_cert
-    cd renew_cert
-    virtualenv ve
-    source ve/bin/activate
-    pip install -r requirements.txt
+```bash
+git clone https://github.com/neutrinet/renew_cert
+cd renew_cert
+virtualenv ve
```

Comment:
This won't create a python3 virtualenv I believe :x

Reply:
Indeed, I forgot to change that part in the Readme…